handlers/istio_config.go

```diff
@@ -315,9 +318,17 @@ func IstioConfigCreate(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	audit(r, "CREATE on Namespace: "+namespace+" Type: "+objectType+" Subtype: "+objectSubtype+" Object: "+string(body))
 	RespondWithJSON(w, http.StatusOK, createdConfigDetails)
 }
 
 func checkObjectType(objectType string) bool {
 	return business.GetIstioAPI(objectType) != ""
 }
+
+func audit(r *http.Request, message string) {
+	if config.Get().Server.AuditLog {
+		user := r.Header.Get("Kiali-User")
+		log.Infof("AUDIT User [%s] Msg [%s]", user, message)
+	}
+}
```

Comment:
If we are going to start to manage and create audit logs for debug and control I think that we should move this audit to the logs folder. What do yo think?

Reply:
In fact, I would create a method to print the message and have consistency with a Datetime.

Are these only audit logs for authorization with Istio or audit about kiali auth too?

Reply:
The goal of the JIRA is to add additional audit log on write operations.
So, if enabled, it will have an external line with a proper AUDIT keyword for debugging.
I reused the existing log to not write just in plain fmt or to split the logs, I mean, I think the output of the container should be unique, so as this is going to be processed via EFK I guess with the keyword is just enough for the goal.
As we audit only write operations, I put the audit() function there (just for simplicity), but yes, when this need to be propagated into more places, it makes sense to move it.